src/actions/getMintlist.ts

```diff
@@ -12,23 +13,30 @@ const extractIPFSHash = (hay: string) => {
 
 const defaultIpfsGateway = "https://ipfs.io/ipfs/";
 
-export const getMintlist = async (config: Config) => {
+export const getMintlist = async (config: Config, contract?: AirWild2Sale) => {
```

Comment:
I don't see any place where this actually gets passed `contract`.

Could you just get the `AirWild2Sale` contract inside of this function instead of taking it as a parameter?